src/daft-plan/src/physical_ops/actor_pool_project.rs

```diff
@@ -48,12 +46,16 @@ impl ActorPoolProject {
         Ok(ActorPoolProject {
             input,
             projection,
-            resource_request,
             clustering_spec,
             num_actors,
         })
     }
 
+    pub fn resource_request(&self) -> ResourceRequest {
+        // TODO: Grab and merge resource requests across all UDFs
+        todo!();
```

Comment:
Did we want to replace this with call to `get_resource_request`?

Reply:
Yes, good spot thanks!